src/autobot/internal/autobotconfiguration.h

```diff
@@ -28,9 +28,11 @@ class AutobotConfiguration : public IAutobotConfiguration
     AutobotConfiguration() = default;
 
     io::path dataPath() const override;
-    io::path drawDataPath() const override;
 
-    io::path scoreDrawData(const io::path& scorePath) const override;
+    io::path filesPath() const override;
+
+    io::path drawDataPath() const override;
+    io::path fileDrawDataPath(const io::path& filePath) const override;
```

Comment:
the name is really confusing

Probably you'd prefer something like (if I understood you correctly):
`io::path drawDataDir() const
io::path drawDataFilePath(const io::path& fileName) const`

Reply:
The purpose of this method is to create a file path with the drawing data for a given score file.